### ISSUE TITLE
Softwire people cannot log in to use the password reset request functionality

### ISSUE THREAD

**Issue by Krenair** [alphagov/re-request-an-aws-account#101]
They see this:
`You can only sign in with accounts from either digital.cabinet-office.gov.uk or cabinetoffice.gov.uk Google organisations.`

@tlwr @OllieJC I think maybe we can just add them to the list of domains allowed to log in? I don't think they'll be able to do anything sensitive, they'd be able to make requests which we might not choose to grant?

**Comment by OllieJC** [comment]
Good point, I would expect these to come from CO DM or tech poc. Also, AFAIK this will only work if they use G Suite.

**Comment by Krenair** [comment]
They should probably come directly from the user without other people in the middle.
The G-Suite thing may be more of a problem. Do we know if they use G-Suite? I assume we can't ask them to sign up for Google accounts with their work emails if not.
